local_planner/test/test_common.cpp

```diff
@@ -127,32 +133,38 @@ TEST(Common, elevationAnglefromCartesian) {
 
 TEST(Common, elevationAngletoIndex) {
```

Comment:
I don't think you need to test the elevation angle and the azimuth angle to index separately. Check them together, in a single test, so you don't have half-initialized structs everywhere.